NPM lockfile exists.

On push, seeing the following error.

```
$ git push flynn master
Counting objects: 9, done.
Delta compression using up to 8 threads.
Compressing objects: 100% (8/8), done.
Writing objects: 100% (9/9), 897 bytes | 0 bytes/s, done.
Total 9 (delta 5), reused 0 (delta 0)
-----> Building portfolio...
-----> Node.js app detected

       PRO TIP: Specify a node version in package.json
       See https://devcenter.heroku.com/articles/nodejs-support

-----> Defaulting to latest stable node: 0.10.33
-----> Downloading and installing node
-----> Exporting config vars to environment
-----> Installing dependencies
       npm WARN package.json portfolio@0.0.1 No description
       npm WARN package.json portfolio@0.0.1 No repository field.
       npm WARN package.json portfolio@0.0.1 No README data
       npm WARN engine imagemin@2.2.1: wanted: {"node":">=0.10.0","npm":">=2.1.5"} (current: {"node":"0.10.33","npm":"1.4.28"})

       > v8flags@1.0.5 install /tmp/build/node_modules/gulp/node_modules/v8flags
       > node fetch.js

       flags for v8 3.14.5.9 cached.

       > ws@0.5.0 install /tmp/build/node_modules/browser-sync/node_modules/socket.io/node_modules/engine.io/node_modules/ws
       > (node-gyp rebuild 2> builderror.log) || (exit 0)

       make: Entering directory `/tmp/build/node_modules/browser-sync/node_modules/socket.io/node_modules/engine.io/node_modules/ws/build'
       CXX(target) Release/obj.target/bufferutil/src/bufferutil.o
       SOLINK_MODULE(target) Release/obj.target/bufferutil.node
       SOLINK_MODULE(target) Release/obj.target/bufferutil.node: Finished
       COPY Release/bufferutil.node
       CXX(target) Release/obj.target/validation/src/validation.o
       SOLINK_MODULE(target) Release/obj.target/validation.node
       SOLINK_MODULE(target) Release/obj.target/validation.node: Finished
       COPY Release/validation.node
       make: Leaving directory `/tmp/build/node_modules/browser-sync/node_modules/socket.io/node_modules/engine.io/node_modules/ws/build'

       > ws@0.4.31 install /tmp/build/node_modules/browser-sync/node_modules/socket.io/node_modules/socket.io-client/node_modules/engine.io-client/node_modules/ws
       > (node-gyp rebuild 2> builderror.log) || (exit 0)

       make: Entering directory `/tmp/build/node_modules/browser-sync/node_modules/socket.io/node_modules/socket.io-client/node_modules/engine.io-client/node_modules/ws/build'
       CXX(target) Release/obj.target/bufferutil/src/bufferutil.o
       SOLINK_MODULE(target) Release/obj.target/bufferutil.node
       SOLINK_MODULE(target) Release/obj.target/bufferutil.node: Finished
       COPY Release/bufferutil.node
       CXX(target) Release/obj.target/validation/src/validation.o
       SOLINK_MODULE(target) Release/obj.target/validation.node
       SOLINK_MODULE(target) Release/obj.target/validation.node: Finished
       COPY Release/validation.node
       make: Leaving directory `/tmp/build/node_modules/browser-sync/node_modules/socket.io/node_modules/socket.io-client/node_modules/engine.io-client/node_modules/ws/build'

       > optipng-bin@2.0.3 postinstall /tmp/build/node_modules/gulp-imagemin/node_modules/imagemin/node_modules/imagemin-optipng/node_modules/optipng-bin
       > node lib/install.js

       ✔ optipng pre-build test passed successfully


       > jpegtran-bin@2.0.1 postinstall /tmp/build/node_modules/gulp-imagemin/node_modules/imagemin/node_modules/imagemin-jpegtran/node_modules/jpegtran-bin
       > node lib/install.js

       ✔ jpegtran pre-build test passed successfully


       > pngquant-bin@2.0.2 postinstall /tmp/build/node_modules/gulp-imagemin/node_modules/imagemin/node_modules/imagemin-pngquant/node_modules/pngquant-bin
       > node lib/install.js

       ✔ pngquant pre-build test passed successfully


       > gifsicle@2.0.1 postinstall /tmp/build/node_modules/gulp-imagemin/node_modules/imagemin/node_modules/imagemin-gifsicle/node_modules/gifsicle
       > node lib/install.js

       ✔ gifsicle pre-build test passed successfully

       npm ERR! EEXIST, open '/app/.npm/ea4720da-app-npm-lodash-2-4-1-package-tgz.lock'
       File exists: /app/.npm/ea4720da-app-npm-lodash-2-4-1-package-tgz.lock
       Move it away, and try again.

       npm ERR! System Linux 3.13.0-24-generic
       npm ERR! command "/tmp/build/vendor/node/bin/node" "/tmp/build/vendor/node/bin/npm" "install" "--userconfig" "/tmp/build/.npmrc" "--production"
       npm ERR! cwd /tmp/build
       npm ERR! node -v v0.10.33
       npm ERR! npm -v 1.4.28
       npm ERR! path /app/.npm/ea4720da-app-npm-lodash-2-4-1-package-tgz.lock
       npm ERR! code EEXIST
       npm ERR! errno 47
       npm ERR! not ok code 0
remote: Build failed: exec: job exited with status 1
To ssh://git@example.io/portfolio.git
 ! [remote rejected] master -> master (pre-receive hook declined)
```

--- a/slugbuilder/builder/buildpacks.txt
+++ b/slugbuilder/builder/buildpacks.txt
@@ -1,6 +1,6 @@
 https://github.com/heroku/heroku-buildpack-multi.git#cddec34c
 https://github.com/heroku/heroku-buildpack-ruby.git#v129
-https://github.com/heroku/heroku-buildpack-nodejs.git#v60
+https://github.com/heroku/heroku-buildpack-nodejs.git#fae41898
 https://github.com/heroku/heroku-buildpack-clojure.git#eb4ff18a
 https://github.com/heroku/heroku-buildpack-python.git#a51c1624
 https://github.com/heroku/heroku-buildpack-java.git#83549077

